packages/dtx-web/src/lib/services/soundLibrary.ts

```
@@ -0,0 +1,308 @@
+export class SoundLibrary {
+	private static readonly STORAGE_KEY = 'dtx_sound_library';
+	private static readonly MAX_STORAGE_SIZE = 50 * 1024 * 1024; // 50MB limit
+	private static readonly LARGE_FILE_THRESHOLD = 2 * 1024 * 1024; // 2MB threshold for large files
+	private static memoryFiles: Map<string, SoundLibraryFile> = new Map(); // In-memory storage for large files
+
+	/**
+	 * Get all files in the sound library (both localStorage and memory)
+	 */
+	static getAll(): SoundLibraryFile[] {
+		try {
+			const data = localStorage.getItem(this.STORAGE_KEY);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)